packages/calcite-components/src/components/stepper/stepper.stories.ts

```diff
@@ -210,3 +210,87 @@ export const verticalLayout_TestOnly = (): string => html`<calcite-stepper layou
       >Step 1 Content Goes Here</calcite-stepper-item
     >
   </calcite-stepper>`;
+
+const getBreakpointsStoryTemplate = ({ scale, width }: { scale: string; width: number }): string => html`<style>
+    calcite-stepper {
+      width: ${width}px;
+    }
+  </style>
+  <calcite-stepper numbered icon layout="horizontal" scale="${scale}">
+    <calcite-stepper-item
+      heading="Choose method"
+      description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua"
+      complete
+      disabled
+    >
+      <calcite-notice open width="full">
+        <div slot="message">Step 1 Content Goes Here</div>
+      </calcite-notice>
+    </calcite-stepper-item>
+    <calcite-stepper-item
+      heading="Compile member list"
+      description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua"
+      complete
+    >
+      <calcite-notice open width="full">
+        <div slot="message">Step 2 Content Goes Here</div>
+      </calcite-notice>
+    </calcite-stepper-item>
+    <calcite-stepper-item
+      heading="Set member properties"
+      description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua"
+      description="Some subtext"
+      error
+    >
+      <calcite-notice open width="full">
+        <div slot="message">Step 3 Content Goes Here</div>
+      </calcite-notice>
+    </calcite-stepper-item>
+    <calcite-stepper-item
+      heading="Confirm and complete"
+      description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua"
+    >
+      <calcite-notice open width="full">
+        <div slot="message">Step 4 Content Goes Here</div>
+      </calcite-notice>
+    </calcite-stepper-item>
+  </calcite-stepper>`;
+
+export const responsiveLayoutScaleS_TestOnly = (): string => getBreakpointsStoryTemplate({ scale: "s", width: 400 });
```

Comment:
Could you use the [`createBreakpointStories` util](https://github.com/Esri/calcite-components/blob/0ee6c16e2e9e3f45185f06eb87705b98bc87aba8/packages/calcite-components/.storybook/utils.tsx#L153-L152) for this? 

BTW, what's the reason behind testing widths between the defined max-width breakpoints?

Reply:
Stepper responsive layout is designed a bit different compared to others. The component doesn't depend upon a fixed breakpoint width and uses min-width to calculate how many items can fit within the page dimensions. The [createBreakpointStories util](https://github.com/Esri/calcite-components/blob/0ee6c16e2e9e3f45185f06eb87705b98bc87aba8/packages/calcite-components/.storybook/utils.tsx#L153-L152) is using hard-corded value to test which has the lowest breakpoint at `476px`. This works currently since we are using 4 `stepper-item's` in each story and the lowest possible width is `480px`, but if there is a change to the value for `xsmall` width the tests will break. 

I don't think changing the value of `xsmall` should break these stories. Perhaps we can enhance the util to add a custom width scale. Thoughts?